src/server/core/readSsl.ts

```diff
@@ -1,11 +1,20 @@
 import { readFileSync } from 'fs';
 import { SslConfiguration } from '../types';
 
+function fileReader(path: string) {
+  try {
+    return readFileSync(path);
+  } catch (e) {
+    // try to read file from node_modules
+    return readFileSync(require.resolve(path));
```

Comment:
Yeah this is great - sure it could have crashed beforehand, too.

To preserve the old error message could we do

```
try {
  path = require.resolve(path);
} catch {}

return readFileSync(path);
```

This way, we'd take the one from node_modules first, but if that cannot be resolved then we'd gracefully fall back to the old one.

Should we adjust the default settings for the key and cert values? I'd might be more robust with the `kras/.../ ` path.

Reply:
Yes, your suggestion is perfect!